configmap/configmap.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
```

Comment:
How about this structure:

* `hello` - first iteration, with Dockerfile
* `hello-k8s` - as `hello`, but with k8s yaml
* `hello-helm` - as `hello-k8s`, but with Helm chart
* `hello-config` - as `hello-helm`, but with ConfigMap
* `hello-secret` - as `hello-config`, but with Secret
* ...
* Profit